src/main/resources/db/changelog/changes/v1.0.0/sql/2021-07-01-02__create_not_available_item_status_table.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE not_available_item_status
+(
+    item_contribution_options_configuration_id UUID NOT NULL,
+    item_status VARCHAR(255) NOT NULL CHECK(item_status NOT IN ('Available', 'Checked out', 'Paged', 'In transit', 'Declared lost', 'Withdrawn', 'Awaiting pickup', 'Awaiting')),
+    PRIMARY KEY (item_contribution_options_configuration_id, item_status),
```

Comment:
Please used named constraints, like
`CONSTRAINT pk_location_mapping PRIMARY KEY (id),`

This rules applies to all constraints (PK, Foreing Key, CHECK etc ) but `NOT NULL`
Take a look at other scripts for examples